src/lib.rs

```diff
@@ -201,7 +201,7 @@ The `Ready` state contains quite a bit of information, and you need to check and
 by one:
 
 1. Check whether `messages` is empty or not. If not, it means that the node will send messages to
-other nodes:
+    other nodes:
```

Comment:
unnecessary space?

Reply:
> unnecessary space?

This is a compiler error from CI, so I made this change.

CI output:
```
error: doc list item missing indentation
   --> src/lib.rs:204:1
    |
204 | other nodes:
    | ^
    |
    = help: if this is supposed to be its own paragraph, add a blank line
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
note: the lint level is defined here
   --> src/lib.rs:478:9
    |
478 | #![deny(clippy::all)]
    |         ^^^^^^^^^^^
    = note: `#[deny(clippy::doc_lazy_continuation)]` implied by `#[deny(clippy::all)]`
help: indent this line
    |
204 |    other nodes:
    | +++
```